tests/conftest.py

```diff
@@ -174,98 +216,172 @@ def standard_image_key_path():
     return "/entry1/tomo_entry/instrument/detector/image_key"
 
 
-@pytest.fixture
-def testing_pipeline():
-    return "tests/samples/pipeline_template_examples/testing/testing_pipeline.yaml"
-
-
+# TODO: depricate when loader is generalised (big data tests instead)
 @pytest.fixture
 def diad_data():
     return "tests/test_data/k11_diad/k11-18014.nxs"
 
 
+# TODO: depricate when loader is generalised
 @pytest.fixture
 def diad_loader():
     return "tests/samples/loader_configs/diad.yaml"
 
 
 @pytest.fixture
-def diad_pipeline_gpu():
-    return "tests/samples/pipeline_template_examples/DLS/01_diad_pipeline_gpu.yaml"
+def standard_loader():
+    return "tests/samples/loader_configs/standard_tomo.yaml"
 
 
 @pytest.fixture
-def i12_data():
-    return "tests/test_data/i12/separate_flats_darks/i12_dynamic_start_stop180.nxs"
+def sample_pipelines():
+    return "tests/samples/pipeline_template_examples/"
+
+
+###########Auto-generated pipelines##################
 
 
 @pytest.fixture
-def pipeline360():
-    return "samples/pipeline_template_examples/DLS/02_i12_360scan_pipeline.yaml"
+def cpu_pipeline_gridrec():
+    return "docs/source/pipelines_full/cpu_pipeline_gridrec.yaml"
 
 
 @pytest.fixture
-def i12_loader():
-    return (
-        "tests/samples/pipeline_template_examples/DLS/03_i12_separate_darks_flats.yaml"
-    )
+def gpu_pipelineFBP():
+    return "docs/source/pipelines_full/gpu_pipelineFBP.yaml"
 
 
 @pytest.fixture
-def i12_loader_ignore_darks_flats():
-    return "tests/samples/pipeline_template_examples/DLS/04_i12_ignore_darks_flats.yaml"
+def gpu_pipelineFBP_denoising():
+    return "docs/source/pipelines_full/gpu_pipelineFBP_denoising.yaml"
 
 
 @pytest.fixture
-def standard_loader():
-    return "tests/samples/loader_configs/standard_tomo.yaml"
+def gpu_pipeline_diad_FBP_noimagesaving():
+    return "docs/source/pipelines_full/gpu_diad_FBP_noimagesaving.yaml"
 
 
 @pytest.fixture
-def sample_pipelines():
-    return "tests/samples/pipeline_template_examples/"
+def gpu_pipeline_diad_FBP():
+    return "docs/source/pipelines_full/gpu_diad_FBP.yaml"
+
+
+@pytest.fixture
+def gpu_pipeline_360_paganin_FBP():
+    return "docs/source/pipelines_full/gpu_360_paganin_FBP.yaml"
+
+
+# ---------------------END------------------------#
+
+###########Raw projection data (big)##################
+# Note that raw_data folder should exist with the datasets
+# listed bellow
```

Comment:
I think this code comment could do with a bit more description about the existence of the `tests/test_data/raw_data` directory.

I believe this is referring to the fact that the following fixtures are only used in Jenkins tests, and that when those tests are executed on the Jenkins machine there's some mechanism which will download extra datasets and put them in the `raw_data/` subdirectory?

If someone were to just look at this code comment, I'd imagine they would look in the `tests/test_data/` directory in the repo and see that there's no `raw_data/` subdirectory, causing confusion.